Week3/homework/step3.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 
 function createBase(base) {
-  // Replace this comment and the next line with your code
-  console.log(base);
+  function sumUp(val) {
+    return base + val;
+  }
+  return sumUp;
 }
 
 const addSix = createBase(6);
```

Comment:
You can also return an anonymous function with using an intermediate variable:

```js
function createBase(base) {
  return function(val) {
    return base + val;
  };
}
```

Or, if you are adventurous, use arrow functions:

```js
const createBase = base => val => base + val;
```